nrovalidate/main.go

```diff
@@ -100,12 +101,12 @@ func validateCluster(args ProgArgs) error {
 		return err
 	}
 
-	data, err := nrovalidator.CollectData(context.TODO(), cli)
+	data, err := nrovalidator.Collect(context.TODO(), cli)
 	if err != nil {
 		return err
 	}
 
-	result, err := nrovalidator.Validate(*data)
+	result, err := nrovalidator.Validate(data)
```

Comment:
nrovalidator.Collect sets the nodes names and the version only, we should also call CollectKubeletConfig() before Validate

Reply:
please check https://github.com/openshift-kni/numaresources-operator/pull/386/commits/464fc94c0db25f404551c8263739d73c3cf322a2#diff-40da5a0178b34c207217154cc7fa3d432882f52681d9e262e33bce27f0156b36R84

Reply:
Or also https://github.com/openshift-kni/numaresources-operator/pull/386/files/bff1ad7f297ca3bc4173540a252778a1f50086cc#diff-40da5a0178b34c207217154cc7fa3d432882f52681d9e262e33bce27f0156b36R80

Reply:
I see, thanks for pointing that!